test/integration/reffy-reports.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
```

Comment:
Just out of curiosity: is it recommended to have the full LICENSE at the beginning of every file? I thought one top-level LICENSE file + one-line copyright header would do.

Reply:
It's what was in the quickstart repo that I started this from. The short form for the existing ones I know of are with BSD licenses, so I'd have to look into how to adapt this to that form. So I didn't :)